projects/plugins/crm/src/automation/class-recipe-exception.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Automattic\Jetpack\CRM\Automation;
+
+class Workflow_Exception extends \Exception {
+    const INVALID_RECIPE                = 10;
+	const RECIPE_REQUIRE_A_TRIGGER      = 11;
+	const RECIPE_REQUIRE_A_INITIAL_STEP = 12;
```

Comment:
These constants wasn't refactored to `workflow`

Reply:
Thanks

Reply:
and the file name either... :)